examples/line/present.py

```diff
@@ -0,0 +1,48 @@
+"""
+Line Plot
+============
+
+Example showing present feature for lines.
+"""
+
+# test_example = true
+
+from fastplotlib import Plot
+import numpy as np
+
+from wgpu.gui.offscreen import WgpuCanvas
+from pygfx import WgpuRenderer
+
+canvas = WgpuCanvas()
+renderer = WgpuRenderer(canvas)
+
+plot = Plot(canvas=canvas, renderer=renderer)
+
+sine = np.load("../data/sine.npy")
+cosine = np.load("../data/cosine.npy")
+sinc = np.load("../data/sinc.npy")
```

Comment:
just make these, don't save them since it's not randomly generated

Reply:
when I didn't save them was when I kept getting errors locally about the data changing...idk what the precision is like when you call `np.sin()` or `np.cos()` on a linspace but I think this is what was causing the problem before and that is why I saved them

Reply:
what if you force `float32` when making the linspace?

Reply:
will try